docs/tasks.md

```diff
@@ -153,3 +139,172 @@ work has completed, spinning up replacement workers.
+
+You can read more about [Ansible Tower Capacity Determination and Job Impact](https://docs.ansible.com/ansible-tower/latest/html/userguide/jobs.html#at-capacity-determination-and-job-impact) in the Jobs section of the Ansible Tower User Guide.
+
+
+### Periodic Background Tasks
+
+Generally speaking, these are the tasks which take up a lot of resources which are best for _not_ running via HTTP request.
+
+
+#### User-Defined Schedules
+
+While jobs can be launched manually in the AWX interface, it's also possible to configure jobs to run automatically on a schedule (such as daily, or every Monday at 9AM). A special background task, `awx_periodic_scheduler`, runs periodically and determines which jobs are ready to be launched.
+
+
+#### Update/Delete Inventory Computed Fields
+
+When making changes to inventories or hosts, there are related attributes that are calculated "under the hood".  For example, when a user runs an inventory update (refer to the "Run Inventory Update" section above for more details) and one of the hosts is offline, this can be a very resource-intensive thing to calculate.  This particular background task can do the work instead!  
+
+In addition to freeing up resources, a handler and wrapper around `inventory.update_computed_fields` get signaled within this task in order to prevent unnecessary recursive calls.
+
+
+#### Update Host Smart Inventory Memberships
+
+The `smart_inventories` field in AWX/Tower uses a membership lookup table that identifies the set of every Smart Inventory a host is associated with. This particular task generates memberships and is launched whenever certain conditions are met (_e.g._, a new host is added or an existing host is modified).
+
+An important thing to note is that this task is only run if the `AWX_REBUILD_SMART_MEMBERSHIP` is set to `True` (default is `False`).
+
+For more information, visit the [Smart Inventories section](https://docs.ansible.com/ansible-tower/latest/html/userguide/inventories.html#smart-inventories) of the Tower User Guide's "Inventory" page or the AWX documentation page [Inventory Refresh Overview page](https://github.com/ansible/awx/blob/devel/docs/inventory_refresh.md#inventory-changes) in this repo.
+
+
+#### Deep Copy Model Object
+
+As previously discussed, there are a number of places where tasks run in the background due to slow processing time or high amounts of memory consumption (_e.g._, in cases where nested code is involved). Since it would be suboptimal to have resource-intensive code running only to have the HTTP connection hanging, this task instead acquires all of the attributes of the original job or object, checks that the user has permissions to it, and then constructs a new compound job/object, recursively adding these copies into the original. This enables a response of a `202 Accepted`, where instead of waiting for a `200 OK` status, it simply indicates that the job is in progress, freeing up resources for other tasks.
```

Comment:
> and then constructs a new compound job/object

You don't copy jobs. You copy objects, you launch jobs.

When we copy things like workflow job templates or inventories, we _do_ create the object within the HTTP request, but we copy the _related_ objects such as hosts and groups within a task.